Dockerfile

```diff
@@ -9,7 +9,7 @@ COPY . /rawhash2
 
 RUN mkdir -p build && cd build \
     && cmake .. \
-    && make -j 3
+    && make -j 2

```

Comment:
Workflows are pretty sensitive to the number of cores. Running on 3 cores is sufficient for cached builds, but without caching they tend to hang (due to increased resource usage?). Running on 2 cores should hopefully work for un-cached builds too, but maybe it's suboptimal to specify the number of cores in Dockerfile itself, rather than provide them as some kind of parameter from workflow...

Reply:
Yes, I would do `make -j $RAWHASH_NUM_BUILD_JOBS`
So the CI would set `RAWHASH_NUM_BUILD_JOBS`, whereas usually, it is unset